Assets/Scripts/Enemies/TankMeleeEnemy.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TankMeleeEnemy : MeleeEnemy
+{
+    void Start()
+    {
+        hp = 60;
+        speed = 0.8f;
+        attackRange = 0.5f;
+        damage = 10;
+        base.Start();
+        ChangeState(new Chase());
```

Comment:
MeleeEnemy already does this when you call base.Start() I believe so this may be duplicate changing the state.